client/src/components/DatasetProjectDataFormat.js

```diff
@@ -1,14 +1,24 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { Select } from 'grommet'
 import { config } from 'config'
+import { useDatasetManager } from 'hooks/useDatasetManager'
 import { getReadableOptions } from 'helpers/getReadableOptions'
 import { FormField } from 'components/FormField'
 import { HelpLink } from 'components/HelpLink'
 
-export const DatasetProjectDataFormat = () => {
-  // NOTE: The dataset endpoint should include all available format options fields per resource
-  const formatOptions = ['SINGLE_CELL_EXPERIMENT', 'ANN_DATA']
-  const [format, setFormat] = useState(formatOptions[0])
+export const DatasetProjectDataFormat = ({
+  project,
+  format,
+  handleSetFormat
```

Comment:
```suggestion
  onFormatChange
```

Reply:
🗒️ We've decided to define the `format` state within this component (instead of the parent) and no longer require the `onFormatChange` prop.